lib/evss/auth_headers.rb

```diff
@@ -19,6 +19,7 @@ def to_h
         'va_eauth_firstName' => @user.first_name,
         'va_eauth_lastName' => @user.last_name,
         'va_eauth_issueinstant' => @user.last_signed_in.iso8601,
+        'va_eauth_birlsfilenumber' => @user.birls_id,
```

Comment:
Is it OK to just add whatever we want to VAFI auth headers?

Reply:
It's an expected header, I can double check with CSRA but it was their suggestion to add it :)

Reply:
So this was expected and we were not previously sending it?  LGTM so long as coordination occurs with CSRA about the VAFI header contract.

Reply:
yes, good point! and I see there's a conversation on #evss asking them to confirm that nothing will explode if we start to send this, and in particular if the value is nil, so I'd wait to merge this until Syed answers that question

Reply:
got confirmation from Anand Sastry
> It is ok to include it. Just like PID, BIRLS number is a correlated identifier and not an enterprise identifier. It can be nil.